content/efcore/getting-started/samples/shared/LibraryContext.cs

```diff
@@ -3,10 +3,11 @@
 
 public class LibraryContext : DbContext
```

Comment:
There is another LibraryContext.cs that needs to be updated as well:
`content\efcore\getting-started\samples\manipulating-entities\shared\LibraryContext.cs`